test/benchmarks/material/button-harness/testbed-benchmark-utilities.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Records the performance of the given function.
+ *
+ * @param id A unique identifier.
```

Comment:
We should make clear for what this unique identifier is used (it's clear to me; but I find the param only describing what; not why)